apps.yaml

```diff
@@ -262,4 +262,40 @@ apps:
     tls:
       key_vault_name: "acmedtscftsboxintsvc"
       name: "wildcard-hmcts-net"
+    internalUrl: "https://backstage-sandbox.hmcts.net/"
```

Comment:
This doesn't seem right - there's an addition of backstage here but it's still defined below - and nothing for the panorama entry?